services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/samples/KibanaSpec.groovy

```diff
@@ -17,21 +17,21 @@ package org.openkilda.functionaltests.spec.samples
 
 import org.openkilda.functionaltests.BaseSpecification
 import org.openkilda.functionaltests.helpers.ElasticHelper
-import org.openkilda.functionaltests.helpers.ElasticQueryBuilder
 import org.springframework.beans.factory.annotation.Autowired
 import spock.lang.Ignore
 
 @Ignore("This is an example specification")
 class KibanaSpec extends BaseSpecification {
+
     @Autowired
-    ElasticHelper kibanaHelper
+    ElasticHelper elasticHelper
 
     def "Test framework should be able to extract logs from Storm Worker"() {
         when: "Kibana Helper is initialized"
-        assert kibanaHelper
+        assert elasticHelper
 
         then: "STORM would generate at least 1 INFO message per 5 minutes"
-        def logs = kibanaHelper.getLogs(ElasticQueryBuilder.buildQuery().addAppId("storm-worker_log").addTimeRange(300))
+        def logs = elasticHelper.buildQuery().addAppId("storm-worker_log").addTimeRange(300).getLogs()
```

Comment:
I understand your idea, but this pattern looks strange for me, especially the buildQuery() method.
I would expect one of 
static builder method: 
`elastic.getLogs(ElasticQuery.builder().appId("sth").tags("sth").level("sth").build())` 
same but with 'with''prefix:
`elastic.getLogs(ElasticQuery.builder().withAppId("sth").withTags("sth").withLevel("sth").build())`
just creating builder directly:
`elastic.getLogs(new ElasticQuery.Builder().withAppId("sth").withTags("sth").withLevel("sth").build())`
creating query directly via map constructor (no builder):
`elastic.getLogs(new ElasticQuery(appId: "sth", tags: "sth", level: "sth"))`

In all cases getLogs takes 'ElasticQuery' as input (NOT builder).
For builder implementation you'll only need:
```
@Builder
public class ElasticQuery {
 String appId, tags, level
}
```
For map constructor basically the same but with another annotation:
```
@MapConstructor //or @Canonical
public class ElasticQuery {
 String appId, tags, level
}
```
 I'm don't have a very strong opinion on changing current implementation, just discussing